UEFITool/uefitool.pro

```diff
@@ -123,7 +123,5 @@ FORMS += uefitool.ui \
 RESOURCES += uefitool.qrc
 RC_FILE = uefitool.rc
 ICON = icons/uefitool.icns
-ICONFILE.files = icons/uefitool.icns
-ICONFILE.path = Contents/Resources
```

Comment:
AFAIR, this was for Qt4, but we don't really support it, so I'm OK with this change.